### PR TITLE
Servos

### PR DESCRIPTION
**Pull request by ROBPEP19**
My name is Rob and I made all the servo stuff. Merge this branch into main, please

### CODE REVIEW

File: lib/servos/src/Servos.cpp [hunk 1]
```diff
@@ -1,59 +1,48 @@
...
 #include <Servo.h>
 #include <servos.h>
 
 Servo servoRight;
 Servo servoLeft;
 
-float lastAngleRight = 0;
-float lastAngleLeft = 0;
+int lastAngleRight = 0;
+int lastAngleLeft = 0;
```
Comment (Arnau478) [inline lib/servos/src/Servos.cpp:15]: Is the angle now stored in degrees? I thought it was about radians...

File: lib/servos/src/Servos.cpp [hunk 2]
```diff
@@ -1,59 +1,48 @@
...
-servoLeft.attach(9);  //*Poner pin servo izq.
-
-servoRight.write(0);
-servoLeft.write(0);
-
-*/
 int velocity = 0;
 
+int _rightPin;
+int _leftPin;
```
Comment (Arnau478) [inline lib/servos/src/Servos.cpp:26]: int variables should always be initialized on definition

```suggestion
int _rightPin = 0;
int _leftPin = 0;
```

File: src/main.cpp [hunk 3]
```diff
@@ -55,4 +57,67 @@ void loop(){
...
+void moveServos(){
+    float _direction;
+    calculate_direction(&_direction, 0 /*latitud de donde quieres ir */, 0 /*longitud de donde quieres ir */,
+    0 /*altura de donde quieres ir */, gps_position()->x, gps_position()->y, gps_position()->z);
+
+    float __direction = map(_direction, 0, 2 * PI, -50, 50) ;
+
+    servo.angleRight(50 - __direction);
+    servo.angleLeft(50 + __direction);
 }
```
Comment (Arnau478) [inline src/main.cpp:123]: Why is there a `_direction` and a `__direction` variable? Please use more concise names for variables. I think naming a variable with 1 or 2 underscores isn't a good practice... Also, in C99, names starting with double-underscore are reserved... Some compilers under some specific configurations wouldn't even compile!
Please fix that...

File: src/main.cpp [hunk 4]
```diff
@@ -55,4 +57,67 @@ void loop(){
...
+
+
+
+
+
+
+
+
+
+
```
Comment (Arnau478) [inline src/main.cpp:110]: Too much spacing...

File: src/main.cpp [hunk 5]
```diff
@@ -55,4 +57,67 @@ void loop(){
...
+}
+
+void loop(){
+    serie = Serial.parseInt();
+    if(serie != 0){
+        angle = serie;
+        Serial.println(angle);
+    }
+    servos_angleLeft(angle);
+
```
Comment (Arnau478) [inline src/main.cpp:88]: Remove this empty line

File: src/main.cpp [hunk 6]
```diff
@@ -55,4 +57,67 @@ void loop(){
...
+/*
+arriba derecha 160
+abajo derecha 30
+
+abajo izquierda 160
+arriba izquierda 25
+
+*/
+int angle = 90;
+int serie;
```
Comment (Arnau478) [inline src/main.cpp:73]: Always initialize integers
```suggestion
int serie = 0;
```

**Review by Arnau478 — COMMENTED** [review]
Not bad, but please fix a few important things I've remarked on my review before we merge. Keep coding!

**Review by Arnau478 — APPROVED** [review]
Well done!

File: lib/servos/src/Servos.cpp (hunk 1 above)
Comment (ROBPEP19) [reply lib/servos/src/Servos.cpp:15]: It is stored in degrees because it stores the position of the servos, not the current orientation of the can.

**Review by salmonmolon — CHANGES_REQUESTED** [review]
Overall just change underscore variables

**Comment by salmonmolon** [comment]
This seems ready to merge

**Review by salmonmolon — APPROVED** [review]

**Pull request merged**